jujugui/static/gui/src/app/components/lightbox/lightbox.js

```diff
@@ -40,8 +160,9 @@ class Lightbox extends React.PureComponent {
 
 Lightbox.propTypes = {
   caption: PropTypes.string,
-  children: PropTypes.any,
-  close: PropTypes.func.isRequired
+  children: PropTypes.node,
```

Comment:
this should be a shape with the allowed types (single and array I gather)

Reply:
I'm not sure how this would be a shape, do you mean `oneOfType`? `node` seems appropriate:
`Anything that can be rendered: numbers, strings, elements or an array (or fragment) containing these types.`